src/web/src/components/NavBar/index.tsx

```diff
@@ -75,6 +75,11 @@ const useStyles = makeStyles((theme) => ({
   },
   avatar: {
     padding: '12px',
+    background: 'none',
```

Comment:
What is this bit doing here?

Reply:
In the Eslint fix PR I made because of configuration, I wrapped every buttons in a `button` element and it created unwanted styles for them. 
This is not related yeah, should I keep it?

Reply:
It's not critical, you can leave it.  But, try not to do this when you can put it in a follow-up.

Reply:
The reason being: anything that will slow down this review should be avoided, it's already a lot.